### PR TITLE
Setting up master redirection file

### PR DESCRIPTION
**Pull request by mairaw**
I've received several broken link bugs for published content that was removed from the repo without setting up redirection. So decided to fix those using the new master redirection file.

If this works, we can also remove other redirected topics currently living in the repo and add them here.

### CODE REVIEW

**Comment by mairaw** [comment]
I've tested all the broken links reported and none of them 404 any longer on the review site. 
@rpetrusha @BillWagner please take a look and let me know if choices for redirects look ok. Thanks!

**Review by rpetrusha — APPROVED** [review]
LGTM.

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        "source_path": "docs/tutorials/getting-started-with-csharp/microservices.md", 
+        "redirect_url": "/dotnet/articles/csharp/tutorials/microservices"
+    },
+    {
+        "source_path": "docs/tutorials/getting-started-with-csharp/working-with-linq.md", 
+        "redirect_url": "/dotnet/articles/csharp/tutorials/working-with-linq"
+    },
+    {
+        "source_path": "docs/tutorials/index.md", 
+        "redirect_url": "/dotnet/articles/csharp/tutorials/"
```
Comment (BillWagner) [inline .openpublishing.redirection.json]: A better destination here would be `/dotnet/articles/samples-and-tutorials/`

**Comment by BillWagner** [comment]
@mairaw Overall, it looks great. I have one suggestion for the tutorial link at the bottom of the file.

**Pull request merged**